<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Accounts Manager - by Yuvraj</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>151</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Insert a new record</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="date">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>92</width>
      <height>22</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <stylestrategy>PreferDefault</stylestrategy>
     </font>
    </property>
    <property name="calendarPopup">
     <bool>true</bool>
    </property>
    <property name="date">
     <date>
      <year>2020</year>
      <month>4</month>
      <day>20</day>
     </date>
    </property>
   </widget>
   <widget class="QComboBox" name="accounts_drop">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Account</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="amount_num">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>104</width>
      <height>22</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Amount (in Rs)</string>
    </property>
   </widget>
   <widget class="QComboBox" name="c_d_drop">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>70</y>
      <width>73</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Credit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Debit</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Credit/Debit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add_button">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>120</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD RECORD</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Create a new account</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select the account name to query details or delete the account</string>
    </property>
   </widget>
   <widget class="QComboBox" name="accounts_drop_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>171</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="query_button">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>240</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Query</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_button">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>240</y>
      <width>71</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_rec_button">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete last record</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
